.github/workflows/main.yml

```diff
@@ -108,10 +108,6 @@ jobs:
         env:
           PYTHONOPTIMIZE: 2
         run: |
-          # Precompile the py as pyc files
-          python.exe -m compileall src\kolibri\ -b
-          # Remove py source files
-          Get-Childitem -Path src\kolibri\ -Include *.py -File -Recurse | Remove-Item -Force
```

Comment:
I think it would be better to keep compiling but skip the migrations. I think this should work but I haven't tested:

```
python.exe -m compileall src\kolibri\ -b -x '\migrations\[^\]+\.py$'
Get-Childitem -Path src\kolibri\ -Include *.py -Exclude '*\migrations\*.py' -File -Recurse | Remove-Item -Force
```

Could you try that out?

Reply:
Yeah, we added this `compileall` bit back in https://phabricator.endlessm.com/T33627. Did you test this and find that it's not worth keeping? Otherwise I'd love if we can keep it around for now :)

Reply:
It works and simply make the code & environment simple.

Reply:
> I think it would be better to keep compiling but skip the migrations. I think this should work but I haven't tested:
> 
> ```
> python.exe -m compileall src\kolibri\ -b -x '\migrations\[^\]+\.py$'
> ```
It still compile all of the py files.
> ```
> Get-Childitem -Path src\kolibri\ -Include *.py -Exclude '*\migrations\*.py' -File -Recurse | Remove-Item -Force
> ```
Tweaked in the new commit.

So, we may keep both *.py and *.pyc in migrations folders.